web_widget_image_webcam/README.rst

```diff
@@ -0,0 +1,69 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+   :alt: License: LGPL-3
+
+=========================
+Web Widget - Image WebCam
+=========================
+
+This module extends the functionality of the image widget and allows to take snapshots with WebCam.
+
+Configuration
+=============
+
+By default, the module works with all `major browsers
+<https://github.com/jhuckaby/webcamjs#browser-support>`_.
+
+An important note for **Chrome 47+** users - this module only works with websites delivered over SSL / HTTPS.
+Visit this for `more info
+<https://github.com/jhuckaby/webcamjs#important-note-for-chrome-47>`_.
+
+But, If you still want this module to work with websites without SSL / HTTPS.
+Here is the steps to do it easily (Always run in Adobe Flash fallback mode, but it is not desirable).
+
+Set the configuration parameter ``web_widget_image_webcam.flash_fallback_mode`` to ``1``
+
+Its done! Now this module also work with websites without SSL / HTTPS.
+
+Usage
```

Comment:
@Yajo and @elicoidal  i didn't get what are you try to say. so, can you more elaborate on what changes in this you need
Thanks..